src/loc_map/include/kalman_filter/ekf.hpp

```diff
@@ -24,7 +25,8 @@ class ExtendedKalmanFilter {
                                           perceived to be 1 meter away */
 
   Eigen::VectorXf X;                  /**< Expected state vector (localization + mapping) */
-  Eigen::MatrixXf P;                  /**< State covariance matrix */
+  //Eigen::MatrixXf P;                  /**< State covariance matrix */
+  Eigen::SparseMatrix<float> P;
```

Comment:
Small thing: I think that the Sparse Matrix should be documentad as well ad covariance matrix

Reply:
Beyond that, the code compiles, executes and tests are passing. Checked